agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -127,6 +130,9 @@ func TestTaskCleanupDoesNotDeadlock(t *testing.T) {
 		dockerId, err := agent.ResolveTaskDockerID(testTask, "1")
 		require.NoError(t, err, fmt.Sprintf("Cycle %d: Error resolving docker id for container in task", i))
 
+		err = testTask.Stop()
```

Comment:
If that is the case, we should scrap this change and update the windows test to be reflective of the same code path.